internal/internal.go

```diff
@@ -112,12 +112,21 @@ func RandomString(n int) string {
 	return string(bytes)
 }
 
-// GetTLSConfig gets a tls.Config object from the given certs, key, and CA files.
-// you must give the full path to the files.
-// If all files are blank and InsecureSkipVerify=false, returns a nil pointer.
+// Deprecated - use GetClientTLSConfig or GetServerTLSConfig instead.
 func GetTLSConfig(
 	SSLCert, SSLKey, SSLCA string,
 	InsecureSkipVerify bool,
+) (*tls.Config, error) {
+	return GetClientTLSConfig(SSLCert, SSLKey, SSLCA, InsecureSkipVerify)
+}
+
+// GetClientTLSConfig gets a tls.Config object from the given certs, key, and CA files
+// for use with a client.
+// The full path to each file must be provided.
+// Returns a nil pointer if all files are blank and InsecureSkipVerify=false.
+func GetClientTLSConfig(
```

Comment:
Just leave this as GetTLSConfig for now.

Reply:
Done.